test/aws/protocol_tests/rest_json_test.exs

```diff
@@ -50,7 +50,7 @@ defmodule AWS.ProtocolTests.RestJSONTest do
 
         {:ok, body, conn} = Plug.Conn.read_body(conn)
 
-        assert body == "{\"Config\":{\"B\":\"two\",\"A\":\"one\"}}"
+        assert body == "{\"Config\":{\"A\":\"one\",\"B\":\"two\"}}"
```

Comment:
Good! I was afraid that the parsed version was in a different order from the original. I guess it was a problem from Poison.